src/tactic/norm_cast.lean

```diff
@@ -253,6 +253,11 @@ classify_type_test decl <|> trace "classifier failed"
     }
 }
 
+-- lemmas to handle the ≥, > and ≠ operators
```

Comment:
I thought that quite recently we had a bunch of simp-lemmas added to the library that did exactly this. But I don't know exactly where they ended up. @fpvandoorn @semorrison I think you guys were involved in this, right?